symfony/plugins/orangehrmOpenidAuthenticationPlugin/modules/openidauth/actions/openIdCredentialsAction.class.php

```diff
@@ -149,6 +149,8 @@ public function getAuthProviderObj($providerType, $provider, $authProviderDetail
                 break;
             case AuthProviderExtraDetails::GOOGLE_PLUS:
                 $googleAuthProvider = new GoogleAuthProvider();
+                $requestCode=$_GET['code'];
+                $googleAuthProvider->setOption($requestCode);
```

Comment:
may be ['requestcode' => $requestCode ]

Reply:
I change requestCode to Array and assign $_GET values.